### PR TITLE
Remove ConfigMap and bump Minio version

### PR DESCRIPTION
**Pull request by nitisht**
#### What this PR does / why we need it: 
Minio server config file is moved to storage backend, and `mc` is the recommended way to configure Minio as explained here: https://docs.minio.io/docs/minio-server-configuration-guide.html

So, `configMap` for config.json is no longer needed.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by nitisht** [comment]
cc @wlan0

**Review by harshavardhana — APPROVED** [review]

File: stable/minio/templates/statefulset.yaml [hunk 1]
```diff
@@ -35,16 +35,14 @@ spec:
           {{- if .Values.tls.enabled }}
           command: [ "/bin/sh",
           "-ce",
-          "cp /tmp/config.json {{ .Values.configPath }} &&
-          /usr/bin/docker-entrypoint.sh minio -C {{ .Values.configPath }} server
+          "/usr/bin/docker-entrypoint.sh minio -C {{ .Values.configPath }} server
```
Comment (harshavardhana) [inline stable/minio/templates/statefulset.yaml:38]: @nitisht @wlan0 `-C` is deprecated we should move to `--certs-dir` next..

Comment (wlan0) [reply stable/minio/templates/statefulset.yaml:38]: Good to know, I'll test and it make a PR to fix

**Pull request merged**